x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/three_way_diff/final_edit/fields/alert_suppression/suppression_fields_selector.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiFormRow, EuiText } from '@elastic/eui';
+import type { DataViewFieldBase } from '@kbn/es-query';
+import { UseField } from '../../../../../../../../shared_imports';
+import { MultiSelectFieldsAutocomplete } from '../../../../../../../rule_creation_ui/components/multi_select_fields';
+import { SUPPRESSION_FIELDS } from './form_schema';
+import * as i18n from './translations';
+
+interface SuppressionFieldsSelectorProps {
+  suppressibleFieldSpecs: DataViewFieldBase[];
+  disabled?: boolean;
+  disabledText?: string;
+}
+
+export function SuppressionFieldsSelector({
+  suppressibleFieldSpecs,
+  disabled,
+  disabledText,
+}: SuppressionFieldsSelectorProps): JSX.Element {
+  return (
+    <>
```

Comment:
Can fragment wrappers (`<>`) here and around `UseField` be removed? Do they serve any purpose?

Reply:
It was a refactoring artefact.